types/relay-runtime/relay-runtime-tests.tsx

```diff
@@ -120,6 +120,14 @@ const environment = new Environment({
                 break;
         }
     },
+    requiredFieldLogger: (arg) => {
+        if (arg.kind === 'missing_field.log') {
+            console.log(arg.fieldPath, arg.owner);
+        } else {
+            arg.kind; // $ExpectType missing_field.throw
```

Comment:
@alloy is this how I test for a specific string value? Or should I wrap this? Maybe this assertion is pointless?